src/cli/such.c

```diff
@@ -643,6 +651,13 @@ int main(int argc, char* argv[])
     char* pubkey = 0;
     char* cmd = 0;
     char* derived_path = 0;
+    int account = BIP44_FIRST_ACCOUNT_NODE;   /* default account (BIP44_FIRST_ACCOUNT_NODE) */
+    char* change_level = BIP44_CHANGE_EXTERNAL;    /* default external (BIP44_CHANGE_EXTERNAL) */
```

Comment:
Could this be `const char*` as well, due to BIP44_CHANGE_EXTERNAL being read only

Reply:
change_level is set to BIP44_CHANGE_EXTERNAL as a default, but can be assigned through command line argument... https://github.com/edtubbs/libdogecoin/blob/0.1.2-dev-such-slip44/src/cli/such.c#L686